db/migrate/20251107120000_add_linked_rounds_model.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddLinkedRoundsModel < ActiveRecord::Migration[7.2]
+  def change
+    create_table :linked_rounds do |t|
+      t.string :wcif_id
+      t.timestamps
+    end
+
+    add_reference :rounds, :linked_rounds, index: true
```

Comment:
This should also be singular `linked_round` methinks. You can also try `foreign_key: true`